pekko-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClientReceptionist.scala

```diff
@@ -26,7 +26,7 @@ object ClusterClientReceptionist extends ExtensionId[ClusterClientReceptionist]
   override def get(system: ActorSystem): ClusterClientReceptionist =
     super.get(system)
 
-  override def lookup() = ClusterClientReceptionist
+  override def lookup: ExtensionId[_ <: Extension] = ClusterClientReceptionist
```

Comment:
To make this cleaner, you can replace this with ExtensionId[object.type] or in this case ExtensionId[ClusterClientReceptionist.type]

Reply:
I removed the type declaration and that seems to compile